### PR TITLE
Add reference to blog article

### PR DESCRIPTION
**Pull request by Creamers158**
Nice addition to read about the cohorts and how they work. (Good visual/picture about how cohorts by Pranay)

### Description

The article gives a little more insight about the working and the use of cohorts. Good article need reference.

### CODE REVIEW

File: packages/docs/validator-guide/celo-foundation-voting-policy.md [hunk 1]
```diff
@@ -133,4 +133,6 @@ Currently receiving Foundation votes:
 If you would like to keep up-to-date with all the news happening in the Celo community, including validation, node operation and governance, please sign up to our [Celo Signal mailing list here](https://celo.activehosted.com/f/15).
 
 You can add the [Celo Signal public calendar](https://calendar.google.com/calendar/u/0/embed?src=c_9su6ich1uhmetr4ob3sij6kaqs@group.calendar.google.com) as well which has relevant dates.
+
+Additional details and explanation about cohorts can be found here [The-celo-foundation-voting-program](https://medium.com/celoorg/the-celo-foundation-voting-program-7ad9fd2e918d) in the Celo Blog article.
```
Comment (barbaraliau) [inline packages/docs/validator-guide/celo-foundation-voting-policy.md]: Suggested update for better sentence flow:

Additional details and explanation about cohorts and the [Celo Foundation Voting Program](https://medium.com/celoorg/the-celo-foundation-voting-program-7ad9fd2e918d) can be found on the Celo Blog.

**Review by barbaraliau — CHANGES_REQUESTED** [review]
Thanks for opening this PR! Please see comment for requested changes.

**Review by aslawson — APPROVED** [review]
LGTM, thanks for the add!

**Comment by aslawson** [comment]
Oh missed @barbaraliau's comment, please address that first 🙏

**Comment by Creamers158** [comment]
> Oh missed @barbaraliau's comment, please address that first 🙏

What am I missing here :) Think I already changed it here: https://github.com/celo-org/celo-monorepo/pull/8142/commits/2bd2754c77d07b0287c42af74863f6cf3c245ef9

**Pull request closed without merging**